src/test/mempool_tests.cpp

```diff
@@ -203,7 +203,7 @@ BOOST_AUTO_TEST_CASE(MempoolIndexingTest)
 
     CTxMemPool::setEntries setAncestorsCalculated;
     std::string dummy;
-    BOOST_CHECK_EQUAL(pool.CalculateMemPoolAncestors(entry.Fee(2000000LL).FromTx(tx7), setAncestorsCalculated, 100, 1000000, 1000, 1000000, dummy), true);
+    BOOST_CHECK_EQUAL(pool.CalculateMemPoolAncestors(entry.Fee(2'000'000LL).FromTx(tx7), setAncestorsCalculated, CTxMemPool::Limits::NoLimits(), dummy), true);
```

Comment:
Why has test been modified?

If it is required, it would be better to separate such changes into their own commit to ensure that tests pass with no changes in testable code, no?

Reply:
See https://github.com/bitcoin/bitcoin/pull/26103#discussion_r972865346

separate commit would be fine

Reply:
I've split it out into separate commit with glozow's rationale in the commit message.